.mypy/type_coverage.py

```diff
@@ -0,0 +1,51 @@
+"""Verify that minimum type coverage is reached."""
+
+from pathlib import Path
+
+import argparse
+import sys
+
+
+def main(argv=sys.argv) -> None:  # pragma: no cover
+    """Run type coverage check."""
+    parser = argparse.ArgumentParser(
+        usage=("python type_coverage.py coverage=80 file=typecov/linecount.txt \n")
+    )
+    parser.add_argument(
+        "coverage",
+        type=float,
+        metavar="<coverage>",
+        help="Minimum required type coverage.",
+    )
+    parser.add_argument(
+        "file",
+        type=str,
```

Comment:
Better way `type=argparse.FileType('r')`

Reply:
I'm not familiar with argparse.FileType, why is it better?

Reply:
It will eliminate lines 29-31. You won't have to verify if file exists and then open it.

Reply:
Sweet!